### ISSUE TITLE
Add a small example of usage + output

### ISSUE THREAD

**Issue by redox** [algolia/pdf-to-algolia-playground#1]
Would be awesome to include that in the readme.

**Comment by rayrutjes** [comment]
Better?
It is to note that the single available example is not a re-usable script in the sense that it accepts no parameters.
It is mainly a way to showcase how someone could achieve the same.

**Comment by redox** [comment]
> Better?
> It is to note that the single available example is not a re-usable script in the sense that it accepts no parameters.
> It is mainly a way to showcase how someone could achieve the same.

Oh yeah OK, now I get how it works :)

I think it's great if you're writing PHP but harder otherwise. What about extending the tika Docker image, including your PHP code in this docker container and expose from there an API answering JSON instead?

Ultimately I would do:

```
$ docker run -d -p 80:3000 algolia/pdf-extractor
$ curl http://localhost:3000/1/extract -d @myfile.pdf
{
[...]
}
```

**Comment by rayrutjes** [comment]
Yes, that is initially what I intended to do, and I consider the current stage as the first iteration.
The next step contains some complexity. After discussing with some SEs, there are lots of ways to split content of multiple records. So I don't think the splitting should be part of the docker container.

I don't expect people to re-use the current working example unless they do PHP.
However I expect them to understand what's happening so that they can implement their own version.

I think this is just the beginning, thank you for your feedback, I'll make sure to iterate and try to create something more re-usable across different technical stacks.
